compiler/qsc_project/src/fs.rs

```diff
@@ -91,4 +91,8 @@ impl FileSystem for StdFs {
             .collect::<Result<_, _>>()
             .map_err(crate::Error::from)?)
     }
+
+    fn resolve_path(&self, base: &Path, path: &Path) -> miette::Result<PathBuf> {
+        Ok(base.join(path))
```

Comment:
Will this leave behind the `..` and `.` that you want to eliminate?

Reply:
You're right, I need to do that explicitly here. Fixed.

This has no effect in this PR, but will become relevant later when we add relative project references (e.g. `../MyDepeendency`)